package.json

```diff
@@ -22,7 +22,7 @@
   },
   "scripts": {
     "build": "yarn build:components",
-    "build:all": "run-s build:components build:site build:preview mv:preview",
+    "build:all": "run-s build:components build:preview mv:preview",
```

Comment:
Can you pull this out from this PR before we merge? I'd love to keep it scoped to just the *.js addition

Reply:
Sure Done.